psi4/src/psi4/libmints/integral.cc

```diff
@@ -259,6 +259,13 @@ TwoBodyAOInt* IntegralFactory::erf_complement_eri(double omega, int deriv, bool
 #endif
 }
 
+TwoBodyAOInt* IntegralFactory::yukawa_eri(double zeta, int deriv, bool use_shell_pairs, bool needs_exchange) {
+    auto integral_package = Process::environment.options.get_str("INTEGRAL_PACKAGE");
```

Comment:
Modify the function to only use libint2 since no other engine supports it.

Reply:
I currently have the function throw an exception if the integral package is not set to Libint2. Should I just let the Yukawa ERI be created regardless of what the user option is set as?

Reply:
Yes. We will always require libint2 to be installed. If another package is added that supports this integral then this ability can be added back.

Reply:
Fixed this